tests/integration/test_github.py

```diff
@@ -396,12 +396,20 @@ def test_create_release(self):
         Raise the number in `tag` when regenerating the response files.
         (The `tag` has to be unique.)
         """
-        count_before = len(self.hello_world_project.get_releases())
+        releases_before = self.hello_world_project.get_releases()
+        latest_release = releases_before[0].tag_name
```

Comment:
This should do the trick:

```suggestion
        latest_release = self.hello_world_project.get_latest_release()
```

Reply:
Ok, you are checking count difference, so we need to get all releases either way.